### PR TITLE
176 missing dates types in openapi

### LINKED ISSUE #176: Missing Dates types in OpenApi
As a Developer I want support for Date and Date Time types in this plugin. It actually generate the property as a String type instead of a proper Date.

### PR DESCRIPTION
**Pull request by 5uso-sng**
Fixes #176. The proposed solution generates DTOs with Local, Zoned, or Offset datetime objects depending on a SpecFile parameter.

Does this cover all the functionality we need?

### CODE REVIEW

**Review by davidgayoso — COMMENTED** [review]
This changes seems to be all good for me.

**Review by janDuinRod — COMMENTED** [review]
Everything seems to be fine

File: multiapi-engine/src/main/java/com/sngular/api/generator/plugin/asyncapi/util/MapperContentUtil.java [hunk 1]
```diff
@@ -270,7 +270,8 @@ private static SchemaFieldObject processFieldObjectList(
     return fieldObject;
   }
 
-  private static void handleItems(final JsonNode schema, Collection<String> modelToBuildList, SchemaFieldObject fieldObject, final boolean required, JsonNode items) {
+  private static void handleItems(
```
Comment (jemacineiras) [inline multiapi-engine/src/main/java/com/sngular/api/generator/plugin/asyncapi/util/MapperContentUtil.java:273]: I think you should need to review your formatter configuration

File: multiapi-engine/pom.xml
Comment (jemacineiras) [inline multiapi-engine/pom.xml:1]: I´m missing some text about this new functionality in the Readme.

**Review by jemacineiras — CHANGES_REQUESTED** [review]

File: multiapi-engine/src/main/java/com/sngular/api/generator/plugin/asyncapi/util/MapperContentUtil.java (hunk 1 above)
Comment (5uso-sng) [reply multiapi-engine/src/main/java/com/sngular/api/generator/plugin/asyncapi/util/MapperContentUtil.java:273]: I'm pretty sure the autoformatter is set to use the OSS Style, that's just how it handles this case. Deleting the like break and autoformatting again gives the same result

File: multiapi-engine/pom.xml
Comment (5uso-sng) [reply multiapi-engine/pom.xml:1]: Added a row to the OpenAPI SpecFile table. Should it be mentioned elsewhere in the readme as well?

**Review by jemacineiras — APPROVED** [review]

**Review by jemacineiras — DISMISSED** [review]

**Review by jemacineiras — APPROVED** [review]

**Pull request merged**